spacelift/data_azure_integrations.go

```diff
@@ -0,0 +1,130 @@
+package spacelift
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal"
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal/structs"
+)
+
+func dataAzureIntegrations() *schema.Resource {
+	return &schema.Resource{
+		Description: "`spacelift_azure_integrations` represents a list of all the Azure integrations in " +
+			"the Spacelift account.",
+
+		ReadContext: dataAzureIntegrationsRead,
+		Schema: map[string]*schema.Schema{
+			"integrations": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"integration_id": {
+							Type:        schema.TypeString,
+							Description: "Immutable ID of the integration.",
+							Optional:    true,
```

Comment:
Same as per the AWS PR I don't think this should be settable - I'm guessing it's maybe just a copy-paste mistake from the single data source?